Application/Source/Detail/View/DetailViewModel.swift

```diff
@@ -25,6 +25,21 @@ class DetailViewModel: ViewModel, SelectionPresentingViewModel {
         return .empty()
     }
 
+    let content: BehaviorRelay<[Food]> = BehaviorRelay(value: [.beans, .greens, .potatoes, .tomatoes])
+
+    let contentListSeparator: String = ", "
+
+    private(set) lazy var contentsListText: Property<String> = {
+        let initial = ""
+        let observable = content.map { [weak self] contents -> String in
+            guard let self = self else { return initial }
```

Comment:
If we remove the need for the `contentListSeparator` property we can remove this guard.